Lib/enum.py

```diff
@@ -414,6 +529,39 @@ def __new__(metacls, cls, bases, classdict, **kwds):
             if _order_ != enum_class._member_names_:
                 raise TypeError('member order does not match _order_')
         #
+        # remove Flag structures if final class is not a Flag
+        if (
+                Flag is None and cls != 'Flag'
+                or Flag is not None and not issubclass(enum_class, Flag)
+            ):
+            delattr(enum_class, '_boundary_')
+            delattr(enum_class, '_flag_mask_')
+            delattr(enum_class, '_all_bits_')
+            delattr(enum_class, '_inverted_')
+        elif Flag is not None and issubclass(enum_class, Flag):
+            # ensure _all_bits_ is correct and there are no missing flags
+            single_bit_total = 0
+            multi_bit_total = 0
+            for flag in enum_class._member_map_.values():
+                if _is_single_bit(flag._value_):
+                    single_bit_total |= flag._value_
+                else:
+                    # multi-bit flags are considered aliases
+                    multi_bit_total |= flag._value_
+            if enum_class._boundary_ is not KEEP:
+                missed_bits = multi_bit_total & ~single_bit_total
+                missed = []
+                while missed_bits:
+                    i = 2 ** (missed_bits.bit_length() - 1)
+                    missed.append(i)
+                    missed_bits &= ~i
```

Comment:
This pattern is repeated about 3 times, so it would be nice to have an `_iter_bits(val)` utility.

Then `missed` list is not needed and you can use the iterator within `join()` below

Reply:
Done.

Reply:
I also made the same change in `_missing_()`.